pkg/operator/ceph/cluster/controller.go

```diff
@@ -121,7 +121,9 @@ func Add(mgr manager.Manager, context *clusterd.Context, clusterController *Clus
 func newReconciler(mgr manager.Manager, context *clusterd.Context, clusterController *ClusterController) reconcile.Reconciler {
 	// Add the cephv1 scheme to the manager scheme so that the controller knows about it
 	mgrScheme := mgr.GetScheme()
-	cephv1.AddToScheme(mgr.GetScheme())
+	if err := cephv1.AddToScheme(mgr.GetScheme()); err != nil {
+		logger.Errorf("failed to add cephv1 scheme to the manager. %v", err)
```

Comment:
If there is an error calling `AddToScheme()` it really is a bug, so we can panic:
```suggestion
		panic(err)
```